### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by vinkla**
[Exclude non-essential files in .gitattributes.](https://www.reddit.com/r/PHP/comments/2jzp6k/i_dont_need_your_tests_in_my_production)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* text=auto
+
+/doc export-ignore
+/tests export-ignore
```
Comment (m1guelpf) [inline .gitattributes]: Why are we ignoring the tests?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.styleci.yml export-ignore
+/.travis.yml export-ignore
+/phpunit.xml.dist export-ignore
+/README.md export-ignore
```
Comment (m1guelpf) [inline .gitattributes]: And why are we ignoring the README?

File: .gitattributes (hunk 1 above)
Comment (vinkla) [inline .gitattributes]: We don't need the tests directory in production.

File: .gitattributes (hunk 2 above)
Comment (vinkla) [inline .gitattributes]: We don't need the README.md file in production.

**Pull request merged**